### PR TITLE
show created by in client interactions (DEV-164)

### PR DESCRIPTION
**Pull request by Davit-BetterAngels**
DEV-164

if clients page show created by, otherwise for whom created the interaction

### CODE REVIEW

**Comment by johnr54321** [comment]
@Davit-BetterAngels Here are the AC on the jira ticket:
1. InteractionCard in the Interactions screen display client name and photo
2. InteractionCard in the Client Profile screen Interactions tab display created_by name and photo

When I test, I see that criteria 1 fails but criteria 2 passes.

**Comment by shiozakiyuko** [comment]
Hi @Davit-BetterAngels 

I got the same result as John did. Thanks.

**Comment by davelindqvist** [comment]
> `Criteria 1: InteractionCard in the Interactions screen display client name and photo?`

I believe Criteria 1 is asking to have a client name and photo here?
image

Any progress for this?

**Comment by mikefeldberg** [comment]
> > `Criteria 1: InteractionCard in the Interactions screen display client name and photo?`
> 
> I believe Criteria 1 is asking to have a client name and photo here? image
> 
> Any progress on this?

@davelindqvist fyi, this is the interaction summary screen, not the InteractionCard. The InteractionCard is the thing that gets displayed in the list of interactions

**Comment by mikefeldberg** [comment]
@Davit-BetterAngels I updated DEV-164 AC. instead of displaying case manager profile photo (which we don't have and haven't scoped yet), let's just display a placeholder avatar for case managers. added [DEV-723](https://betterangels.atlassian.net/browse/DEV-723) as an eventual follow up to implement actual cm profile photo

[DEV-723]: https://betterangels.atlassian.net/browse/DEV-723?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

**Comment by johnr54321** [comment]
this is now working.  PASS on both criteria

File: libs/expo/betterangels/src/lib/screens/Clients/index.tsx [hunk 1]
```diff
@@ -230,6 +230,7 @@ export default function Clients({ Logo }: { Logo: ElementType }) {
           renderItem={({ item: clientProfile }) =>
             clients ? (
               <ClientCard
+                imageUrl={clientProfile.profilePhoto?.url}
```
Comment (mikefeldberg) [inline libs/expo/betterangels/src/lib/screens/Clients/index.tsx:233]: I think this also needs to be done in `NoteCard`

Comment (Davit-BetterAngels) [reply libs/expo/betterangels/src/lib/screens/Clients/index.tsx:233]: don't see on NoteType under client a profile photo. we can look at it together

**Review by mikefeldberg — APPROVED** [review]
approved with partial AC because backend prereqs missing. created DEV-723 and DEV-736 to cover the missing AC

**Pull request merged**